modelica_builder/transformation.py

```diff
@@ -105,7 +105,7 @@ def make_edits_for_modifications(class_modification_node, modifications, parser,
                     depth=depth + 1,
                     indented=indented)
             else:
-                edit = Edit.make_replace(f'={requested_modification_value}')
+                edit = Edit.make_replace(f'{requested_modification_value}')
```

Comment:
the `=` sign shouldn't be in the replace. This should really be part of the `get_span` method when there is a`ModificationContext`.

Reply:
I think we do want this here, the node we're modifying, `modification`, where we _aren't_ doing a class modification (the else case we are in here) is defined in the grammar like this:
```
modification
   : class_modification ('=' expression)?
   | '=' expression
   | ':=' expression
   ;
```
ie we are overwriting the `=` and the `expression`